Cooper/ClientApp/src.v2/app/layouts/game/game.layout.ts

```diff
@@ -1,10 +1,37 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Game } from '@models';
+import { GamesService } from '@services';
 
 @Component({
   selector: 'coop-game-layout',
   templateUrl: './game.layout.html',
   styleUrls: ['./game.layout.css']
 })
-export class GameLayoutComponent {
+export class GameLayoutComponent implements OnInit {
+  public game: Game;
+  public url: string;
 
+  constructor(private route: ActivatedRoute, private gameService: GamesService) {
+    this.game = {
```

Comment:
I think it will be better to provide common progress indicator (In future). For now I suggest to remove this and use simple check in template whether current game exist [ use *ngIf].